test/local/account_routes.js

```diff
@@ -754,6 +755,9 @@ test('/account/login', function (t) {
     customs: {
       check: function () {
         return P.resolve()
+      },
+      flag: function () {
```

Comment:
This was failing for me locally yesterday without it. Once I `npm update` it works as expect. Removing.